lib/teleterm/clusters/cluster_databases.go

```diff
@@ -131,30 +132,25 @@ func (c *Cluster) reissueDBCerts(ctx context.Context, clusterClient *client.Clus
 		AccessRequests: c.status.ActiveRequests.AccessRequests,
 	})
 	if err != nil {
-		return trace.Wrap(err)
+		return tls.Certificate{}, trace.Wrap(err)
 	}
 
-	// Fetch the certs for the database.
-	err = clusterClient.ReissueUserCerts(ctx, client.CertCacheKeep, client.ReissueParams{
+	key, _, err := clusterClient.IssueUserCertsWithMFA(ctx, client.ReissueParams{
 		RouteToCluster: c.clusterClient.SiteName,
 		RouteToDatabase: proto.RouteToDatabase{
 			ServiceName: routeToDatabase.ServiceName,
 			Protocol:    routeToDatabase.Protocol,
 			Username:    routeToDatabase.Username,
 		},
 		AccessRequests: c.status.ActiveRequests.AccessRequests,
-	})
+		RequesterName:  proto.UserCertsRequest_TSH_DB_LOCAL_PROXY_TUNNEL,
+	}, c.clusterClient.NewMFAPrompt(mfa.WithPromptReasonSessionMFA("database", routeToDatabase.ServiceName)))
```

Comment:
Could you adjust the call to `mfa.WithPromptReasonSessionMFA` in `lib/teleterm/clusters/cluster_apps.go` to pass `"application"` instead of `"Application"`? This way the capitalization is going to be consistent across all three resource types.

Edit: Nvm, you've already thought of it, thanks. :)